content/apt/apt-1/contents.lr

```diff
@@ -58,13 +58,33 @@ Warning symptom, when running sudo apt update:
    Skipping acquire of configured file 'main/binary-i386/Packages' as repository 'http://deb.torproject.org/torproject.org focal InRelease' doesn't support architecture 'i386'
    ```
  
-#### 3. Then add the gpg key used to sign the packages by running the following commands at your command prompt
+#### 3. Then import the gpg key used to sign the packages
 
+   You can download the key directly into `/etc/apt/trusted.gpg.d` either 
+   
+   using **cURL**
```

Comment:
Wget is installed by default on Debian, so I don't see why we need curl instructions.